src/mesh/SX126xInterface.cpp

```diff
@@ -26,20 +26,25 @@ template <typename T> bool SX126xInterface<T>::init()
     pinMode(SX126X_POWER_EN, OUTPUT);
 #endif
 
-#ifndef SX126X_E22
-    float tcxoVoltage = 0; // None - we use an XTAL
-#else
-    // Use DIO3 to power tcxo per https://github.com/jgromes/RadioLib/issues/12#issuecomment-520695575
+// If we need more TCXO voltage ranges in the future we can split this into DIO3_AS_TCXO and DIO3_TCXO_VOLTAGE
+#ifdef DIO3_AS_TCXO_AT_1V8
+    // Used in E22 modules as TCXO voltage is 1.8V per https://www.ebyte.com/en/pdf-down.aspx?id=781 (and https://github.com/jgromes/RadioLib/issues/12#issuecomment-520695575)
+    // (DIO3 is not free to be used as an IRQ)
+    LOG_DEBUG("DIO3_AS_TCXO_AT_1V8 mode true. Using DIO3 to power TCXO with 1.8V\n");
     float tcxoVoltage = 1.8;
+#else
+    // (DIO3 is free to be used as an IRQ)
+    LOG_DEBUG("DIO3_AS_TCXO_AT_1V8 mode false. Not using DIO3 to power TCXO\n");
+    float tcxoVoltage = 0; // None - we use an XTAL
 #endif
     bool useRegulatorLDO = false; // Seems to depend on the connection to pin 9/DCC_SW - if an inductor DCDC?
 
     RadioLibInterface::init();
 
-    if (power == 0)
-        power = SX126X_MAX_POWER;
+    // Do not correct 0dBm to 22dBm as may be using external amplifier! (removed code that did this)
```

Comment:
Don't make this change... we already have precdence for this. Look at the SX126X_MAX_POWER definition in the Station G1 variant for an example. Default limit for SX1262/8 is 22dbm. If targets have amplification, they must set their own limits in the variant

Reply:
That code was for setting sensible defaults when we install a new configuration cause of version changes. Not sure it's still neccessary, it sure was transitioning from 1.2 to 2.0